src/i18n/translations.ts

```diff
@@ -31,8 +31,25 @@ export const no = {
     "gir sponsorer muligheten til å nå en bred målgruppe som inkluderer studenter og etablerte utviklere innen bransjen, under Bergen Open Source-konferansen. Vi ønsker selvsagt å gjøre konferansedagen enda mer minneverdig for dem! Dette kan inkludere alt i fra en kort presentasjon av selskapet til tilpassede logo-merkede sokker og godbiter. (Kanskje til og med noen overraskelser som gleder deltakerne🎫🍺?)",
   "sponsors.contact1": "Ønsker du også å støtte konferansen som sponsor?",
   "sponsors.contact2": "Ta kontakt med Sindre på 📨: ",
-  "sponsors.contact3":
-    "Vennligst merk at vi har et begrenset antall plasser tilgjengelig for lyntaler.",
+  "sponsors.price":"Vi har en sponsorpakke til 5000,- som inneholder følgende ting: ",
+  "sponsors.goods":[
+    {
+      "item":"Vår takknemlighet via SoMe innlegg",
+    },
+    {
+      "item":"Logo på nettside",
+    },
+    {
+      "item":"Roll-up banner på konferanse (sponsor må stille med egen)",
+    },
+    {
+      "item":"Bruke BOSkonf logo selv",
+    },
+    {
+      "item":"Jobbutlysninger kan bli nevnt i introduksjon og avslutning av konferansen (høyst relevant for både studenter og deltakere allerede i jobb)",
+    },
+  ],
```

Comment:
```suggestion
  "sponsors.goods": [
    "Vår takknemlighet via SoMe innlegg",
    "Logo på nettside",
    "Roll-up banner på konferanse (sponsor må stille med egen)",
    "Bruke BOSkonf logo selv",
    "Jobbutlysninger kan bli nevnt i introduksjon og avslutning av konferansen (høyst relevant for både studenter og deltakere allerede i jobb)",
  ],
```

Endre til list of strings og så endrer du typedefinisjonen i utils.ts useTranslation til `return function t(key: keyof typeof no): string | string[] {` på linje 21 så blir ting bygga